### ISSUE TITLE
[Feature Request] NVDA screen reader doesn't read once checked/unchecked on v-checkbox component 

### ISSUE THREAD

**Issue by Ben-SMS** [vuetifyjs/vuetify#12718]
### Problem to solve
When pressing the space bar on the v-checkbox component to checked or unchecked, the NVDA screen reader doesn't read aloud. 
**This only happens on Firefox browser.**

### Proposed solution
NVDA should read once v-checkbox is checked/unchecked so disability users can know they have input or not

**Comment by Ben-SMS** [comment]
Hi John,
I noticed that this bug was marked as closed without any accompanying comments, so I'm unsure whether it has been fixed. Could you please provide more information?

Thanks.
